qldbdriver/qldbdriver.go

```diff
@@ -27,10 +26,10 @@ import (
 
 // DriverOptions can be used to configure the driver during construction.
 type DriverOptions struct {
-	// The amount of times a transaction will automatically retry upon a recoverable error. Default: 4.
-	RetryLimit uint8
+	// The policy guiding retry attempts upon a recoverable error. Default: MaxRetryLimit: 4, SleepBase: 10ms, SleepCap: 5000 ms.
```

Comment:
Nit: SleepCap has an extra space after 5000.

Also, are we not mentioning the default is exponential?